### PR TITLE
Create homepage

### PR DESCRIPTION
**Pull request by auloin**
The website isn't fully responsive but a lot of styling done (```style.css``` is where you'll find most of the interesting stuff).

### CODE REVIEW

File: src/components/Newsletter.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, { useRef } from 'react';
+
+const Newsletter = () => {
+  const input = useRef();
+  const submit = e => {
+    e.preventDefault();
+    // console.log(input.current.value);
```
Comment (jbelien) [inline src/components/Newsletter.jsx:7]: Try to not commit unnecessary comments.
```suggestion

```

**Review by jbelien — CHANGES_REQUESTED** [review]
Some remarks:
- Are we sure all browsers are compatibles with TTF font ?
- Could you make sure all files are linted (should be done with `lint-staged`) ?
- `style.css` file is starting to be quite long, could you split it (I personnaly use SASS but you can use whatever you want) ?
- Could you clean unecessary old Gatsy files (`/content/blog`, for instance) ?
- Could you structure the components folders ? Not an issue at the moment but the more components we have, the more "difficult" it will be to find it if all the components are in the same directory.

Also try to work on a branch with a specific name (`homepage` for this PR for instance) so we can delete the branch when the PR is merged.

Thanks!
Good job so far 👍

**Comment by auloin** [comment]
> * Are we sure all browsers are compatibles with TTF font ?

Support looks good at first glance: [https://caniuse.com/ttf](url)

> * Could you make sure all files are linted (should be done with `lint-staged`) ?

Should be good now (I think only (newly) modified files are checked by lint-staged)

> * `style.css` file is starting to be quite long, could you split it (I personnaly use SASS but you can use whatever you want) ?

This is something I'm thinking about but not sure what's the right way to deal with it: I could work in a component way where every component has its own "stylesheet". How would you like to see me deal with this?

> * Could you clean unecessary old Gatsy files (`/content/blog`, for instance) ?

Done (mostly, I'm keeping some files that may reveal useful).

> * Could you structure the components folders ? Not an issue at the moment but the more components we have, the more "difficult" it will be to find it if all the components are in the same directory.

I'll as soon as I see an opportunity!

**Review by jbelien — APPROVED** [review]

**Comment by jbelien** [comment]
> This is something I'm thinking about but not sure what's the right way to deal with it: I could work in a component way where every component has its own "stylesheet". How would you like to see me deal with this?

I like the idea (that's how Svelte does it). 👌

**Pull request merged**